.github/workflows/build-admin-ui.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build Admin UI
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build Admin UI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        id: node-modules-cache
+        with:
+          path: '**/node_modules'
+          key: node-modules-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            node-modules-
+      - uses: actions/setup-node@v3
```

Comment:
this is not a blocker and can be addressed later. I wonder if we should update our setup-node version [here](https://github.com/hashicorp/boundary-ui/blob/main/.github/workflows/monorepo-validate.yaml#L55) as well?

Reply:
@DhariniJeeva You got it! Nice catch! It was on the toDo list but you are right, there is no reason for it to be included on this PR.
I changed to the V3 in all the actions, they release it when I started testing this 

Nice catch!